src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesArray = sourceString.trim().split(';');
+
+  const stylesObject = {};
+
+  for (const style of stylesArray) {
+    const [property, value] = style.split(':').map((el) => el.trim());
+
+    if (property && value) {
+      stylesObject[property] = value;
+    }
+  }
+
+  return stylesObject;
```

Comment:
The function `convertToObject` uses a `for...of` loop to iterate over the array of styles. According to the checklist, you should avoid using loops and use array iteration methods instead. Consider using methods like `reduce` to transform the array into an object, which would be more in line with functional programming practices.